src/components/JobsTrainingContent/index.scss

```diff
@@ -0,0 +1,48 @@
+@import "../../style/variables";
+@import "../../style/buttons";
+
+.jobsTrainingContent {
+  .searchBar {
+    margin: 3rem 0;
+  }
+
+  .trainingsBtn,
+  .jobsBtn {
```

Comment:
can this just be one class, opportunityListBtn or something like this. unless you need to distinguish the CSS for trainings and jobs for some reason

Reply:
It can be the same class.